### ISSUE TITLE
ember-qunit moduleForComponent 2WB Breakage on Canary

### ISSUE THREAD

**Issue by rwjblue** [emberjs/ember.js#13557]
I have not dug in deeply, but I suspect that all the work around glimmer integration has change/broken the mechanism that ember-test-helpers binds a components context.

Failing test:

```js
test('it renders', function(assert) {
  this.render(hbs`{{input value=value}}`);

  this.$('input').val('1').change();

  assert.equal(this.get('value'), '1');
});
```

The assertion fails (`this.get('value')` returns `undefined`).

Demo: http://rwjblue.jsbin.com/cebadu/edit?html,js,output

/cc @krisselden

**Comment by rwjblue** [comment]
I kinda suspect that https://github.com/switchfly/ember-test-helpers/blob/16a605b1cebc36fbf6afc0a263e7ebdb72bb23da/lib/ember-test-helpers/test-module-for-component.js#L151-L152 are the root issue, but that is just a WAG.

@krisselden - Has anything changed on canary WRT to supporting setting `context` or `controller` on a component?

**Comment by pixelhandler** [comment]
@rwjblue I'm curious about the test 'it renders', does `value` need to be set in the context of the test block?

```
test('it renders', function(assert) {
  this.set('value', '0');
  this.render(`{{input value=value}}`);
  this.$('input').val('1').change();
  assert.equal(this.get('value'), '1');
});
```

Also I've still been using `Ember.run` is that not needed anymore? example: http://jsbin.com/quhuli/4/edit?js,output

**Comment by rwjblue** [comment]
This was fixed by ember-test-helpers@0.5.26.
